driver/main.ml

```diff
@@ -213,7 +213,8 @@ let main () =
           let s = extract_output !output_name in
           if (Filename.check_suffix s Config.ext_obj
             || Filename.check_suffix s Config.ext_dll
-            || Filename.check_suffix s ".c")
+            || Filename.check_suffix s ".c"
+            || Filename.check_suffix s ".exe")
```

Comment:
The ".exe" extension should be added to the error message bellow

Reply:
Thanks, fixed.